WARNING:__main__:
*****************************************
Setting OMP_NUM_THREADS environment variable for each process to be 1 in default, to avoid your system being overloaded, please further tune the variable for optimal performance in your application as needed. 
*****************************************
| distributed init (rank 0)
| distributed init (rank 2)
| distributed init (rank 3)
| distributed init (rank 1)
The model has been loaded.
[2023-11-18 11:22:28,407] torch.distributed.elastic.multiprocessing.api: [WARNING] Sending process 1132671 closing signal SIGTERM
[2023-11-18 11:22:28,415] torch.distributed.elastic.multiprocessing.api: [WARNING] Sending process 1132672 closing signal SIGTERM
[2023-11-18 11:22:28,415] torch.distributed.elastic.multiprocessing.api: [WARNING] Sending process 1132674 closing signal SIGTERM
[2023-11-18 11:22:29,405] torch.distributed.elastic.multiprocessing.api: [ERROR] failed (exitcode: -9) local_rank: 2 (pid: 1132673) of binary: /home/chloezh/miniconda3/envs/torchsrh/bin/python
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/home/chloezh/miniconda3/envs/torchsrh/lib/python3.11/site-packages/torch/distributed/run.py", line 810, in <module>
    main()
  File "/home/chloezh/miniconda3/envs/torchsrh/lib/python3.11/site-packages/torch/distributed/elastic/multiprocessing/errors/__init__.py", line 346, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/chloezh/miniconda3/envs/torchsrh/lib/python3.11/site-packages/torch/distributed/run.py", line 806, in main
    run(args)
  File "/home/chloezh/miniconda3/envs/torchsrh/lib/python3.11/site-packages/torch/distributed/run.py", line 797, in run
    elastic_launch(
  File "/home/chloezh/miniconda3/envs/torchsrh/lib/python3.11/site-packages/torch/distributed/launcher/api.py", line 134, in __call__
    return launch_agent(self._config, self._entrypoint, list(args))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/chloezh/miniconda3/envs/torchsrh/lib/python3.11/site-packages/torch/distributed/launcher/api.py", line 264, in launch_agent
    raise ChildFailedError(
torch.distributed.elastic.multiprocessing.errors.ChildFailedError: 
========================================================
main_dino_ddp.py FAILED
--------------------------------------------------------
Failures:
  <NO_OTHER_FAILURES>
--------------------------------------------------------
Root Cause (first observed failure):
[0]:
  time      : 2023-11-18_11:22:28
  host      : gl1516.arc-ts.umich.edu
  rank      : 2 (local_rank: 2)
  exitcode  : -9 (pid: 1132673)
  error_file: <N/A>
  traceback : Signal 9 (SIGKILL) received by PID 1132673
========================================================
slurmstepd: error: Detected 1 oom_kill event in StepId=65122790.batch. Some of the step tasks have been OOM Killed.
